### ISSUE TITLE
How to load a wizard in the last state.

### ISSUE THREAD

**Issue by redhead** [mgonto/angular-wizard#118]
I am creating a form in a wizard that is saved on each step. The user can go away and then come back to the wizard at later time. How can I get angular wizard to show on startup the step I want (on the one user left) and to set the selected property of steps before as completed?

**Comment by jacobscarter** [comment]
You would have to save the step number somewhere (database, localstorage, wherever you want) and then when your controller loads you would need to check and see if you have a step stored, if you do utilize the `WizardHandler.wizard().goTo(stepNumber);` method to switch to that state.

**Comment by redhead** [comment]
Yes, I've read that in docs. But this way, the steps before the step I want to go to are not set as completed.

**Comment by jacobscarter** [comment]
@redhead  
I have experienced frustration with this issue myself, I think I will add a feature that turns of sequential steps.  So you can bounce around to any step you want in any order.  Would this solve your problem?

**Comment by redhead** [comment]
I guess.

Also, I think it would be awesome to add some API to get and alter states programatically. For example a method `getStep(2)` and be able to change `completed` property of it and such.

Also I ran into problem with triggering action once the wizard has moved to some step, because the inner promise (that `all`s canenter/canexit promises) is never returned, and so disabling me to have a `then` callback on promise resolve.

**Comment by devangnaghera** [comment]
Is this feature added in 1.1.1 ?
